### PR TITLE
upgrade packages and remove AUTHOR_EMAIL and AUTHOR_NAME because it breaks signed commit

### PR DESCRIPTION
**Pull request by Solvik**

### CODE REVIEW

File: gitlab2sentry/utils/gitlab_provider.py [hunk 1]
```diff
@@ -220,7 +216,7 @@ def _get_default_mentions(self, project: Project) -> str:
         return ", ".join(
             [
                 f"@{member.username}"
-                for member in project.members.all()
+                for member in project.members.list(all=True)
```
Comment (Lowaiz) [inline gitlab2sentry/utils/gitlab_provider.py:219]:
```suggestion
                for member in project.members.list(all=True, iterator=True)
```

File: requirements.txt [hunk 2]
```diff
@@ -1,15 +1,7 @@
-aiohttp==3.8.1
+aiohttp==3.10.5
 awesome-slugify==1.6.5
```
[the remark this answers is not in the document]
Comment (Lowaiz) [reply requirements.txt:2]: je parle de `awesome-slugify`

File: Dockerfile [hunk 3]
```diff
@@ -10,6 +10,7 @@ RUN groupadd --gid 1000 appuser \
 
 USER appuser
 COPY gitlab2sentry/ gitlab2sentry/
+COPY tests/ tests/
```
[the remark this answers is not in the document]
Comment (Solvik) [reply Dockerfile:13]: ça fail les imports sinon

File: gitlab2sentry/resources.py [hunk 4]
```diff
@@ -138,16 +136,6 @@ def is_test_env(env: str) -> bool:
         else int(os.environ["GITLAB_GRAPHQL_PAGE_LENGTH"])
     )
     GITLAB_GROUP_IDENTIFIER = os.getenv("GITLAB_GROUP_IDENTIFIER", "")
-    GITLAB_AUTHOR_EMAIL = (
```
[the remark this answers is not in the document]
Comment (Lujeni) [reply gitlab2sentry/resources.py:141]: I agree that it’s important to apply conditions in this case, as it remains relevant in other setups as well. I'm working on proposing a change that aligns with this approach.

**Comment by Lujeni** [comment]
fix via https://github.com/numberly/gitlab2sentry/pull/25

**Pull request closed without merging**